README.md

```diff
@@ -7,8 +7,20 @@ A community-led docker container for RaspAP
 # Usage
 ```
 docker run --name raspap -it -d --privileged --network=host -v /sys/fs/cgroup:/sys/fs/cgroup:ro --cap-add SYS_ADMIN jrcichra/raspap-docker
-docker exec -it raspap bash
-$ ./setup.sh
-docker restart raspap
 Web GUI should be accessible on http://localhost by default
 ```
+## Workaround for arm devices
+To use this container on arm devices you have to make cgroups writable:
+```
+docker run --name raspap -it -d --privileged --network=host --cgroupns=host -v /sys/fs/cgroup:/sys/fs/cgroup:rw --cap-add SYS_ADMIN jrcichra/raspap-docker
```

Comment:
Should we just make the default usage include the writable cgroups and remove this workaround section?

Reply:
i think we should keep it as a workaround. Because this has some serious security consequences. normally with an innocent process this is no harm, but with an accesspoint/vpnclient/simple-router this is not ideal.

https://www.nginx.com/blog/what-are-namespaces-cgroups-how-do-they-work/

this workaround only applies to ARM devices if i am right